unicoremc/views.py

```diff
@@ -38,6 +38,11 @@ def get_repos(refresh):
             break
         repos.extend(data)
         pageNum += 1
+    repos = [{
+        'name': r.get('name'),
+        'git_url': r.get('git_url'),
+        'clone_url': r.get('clone_url')
+    } for r in repos]
```

Comment:
@Rizziepit the data returned from Github was very big (2MB), so I'm instead only caching the fields that we need.